source/blog/2017-08-17-release-notes-7-00.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Finagle 7 Release Notes
+published: true
+post_author:
+  display_name: Daniel Schobel
+  twitter: dschobel
+tags: Releases, Finagle, Finatra, Util, Scrooge, TwitterServer
+---
+
+The August release is here and brings a major version bump for both Finagle and Util to v7! :cake::cake::cake:
+
+Over the last few years Finagle has had an extensive internal overhaul to support Netty 4 along with a rewrite of all its protocols so now as the last traces of Netty 3 have been removed from finagle-core we felt the time was right to celebrate with a shiny new major version.
+
+(Fun fact: Finagle's last major version bump came almost 5 years ago, a simpler time when 'Gangnam Style' was still dominating the charts.)
+
+Without further ado, here are the highlights:
+
+[Finagle 7.0.0][finagle], [Finatra 2.12.0][finatra], [Scrooge 4.19.0][scrooge], [TwitterServer 1.31.0][twitterserver], and [Util 7.0.0][util].
+
+
+### Finagle ###
```

Comment:
I think we should also mention that finagle-stats is no longer depending on Commons (big deal).